indico_toolkit/types/classifications.py

```diff
@@ -0,0 +1,39 @@
+from typing import Dict
+import pandas as pd
+
+from indico_toolkit.pipelines import FileProcessing
+
+
+class Classifications:
+    """
+    Functionality for classification predictions
+    """
+    def __init__(self, predictions: dict):
```

Comment:
should probably just be 'prediction' as there is only one. Likewise, maybe just 'Classification' as class name?

Reply:
same for _pred's' etc. etc.